src/routes/public/policies.js

```diff
@@ -26,7 +29,11 @@ exports.register = function (server, options, next) {
       validate: {
         headers: Joi.object({
           'authorization': Joi.any().required()
-        }).unknown()
+        }).unknown(),
+        query: Joi.object({
+          page: Joi.number().integer().positive().required().description('Page number, starts from 1'),
+          limit: Joi.number().integer().positive().required().description('Items per page')
```

Comment:
checking again this PR and @mihaidma 's [one](https://github.com/nearform/labs-authorization/pull/277/files#diff-d9b6925c9c7594c6a87759aadcb0fe8bR452) I've the feeling that make the query string parameters required is not correct.

From my experience, if we need to require a certain parameter, we should put it in the URL (ie `/organizations/{page}/{limit}`). On the other hand, if we are dealing with optional parameters, we may want to put them in the query string.

I think we should have a default for `page` and `limit` params, and for those calls without them we return the result for `page=1` and `limit=X`